crates/cli/src/args.rs

```diff
@@ -17,7 +20,7 @@ use tracing::metadata::Level;
 #[command(author, version, about, long_about = None)]
 pub struct Args {
     /// The input format, if not provided will be inferred from the input file's extension, falling back to json
-    #[arg(short, long, global = true)]
+    #[arg(short, long, global = true, value_parser=PossibleValuesParser::new(["json", "json-pretty", "ndjson", "geoparquet", "geoparquet[snappy]"]).try_map(|s| Format::from_str(&s[..])))]
```

Comment:
This is one way, list all possible options as strings, use `Format` and pass them on to `Format::from_str`